torch_xla/csrc/ops/eye.cpp

```diff
@@ -0,0 +1,33 @@
+#include "torch_xla/csrc/ops/eye.h"
```

Comment:
This needs not to be a ops/FOO.* thing.
This can be an ops.* FOO thing.
The only arguments describe a shape, so this can be a GenericOp within ops.*
Let's call this Identity().

Reply:
The way it's done this way is because it needs access to the builder since it doesn't have any tensor inputs. What is your suggestion here, have the helper in ops take a builder? It looked weird, which is why I did it this way.

Reply:
The loctx->builder() is what you need ;)

Reply:
Sure, that means passing `loctx` instead, which still looks weird, but somewhat acceptable. Is this your suggestion?

Reply:
You do not pass anything.
In the lowering function that you pass to GenericOp, you call xla::IdentityMatrix() with loctx->builder().
The loctx comes as lowering function argument.

Reply:
Just to be clear, there will be no more eye.h/cpp.
Everything is in 10 lines in ops.cpp

Reply:
Ok, I'll do it that way. Didn't quite like the `GenericOp` route because the lines and columns parameters, but I guess we can bend the rules a bit here.

Reply:
Yes, usually when we have "arguments" to the op, we use ops/FOO.*
In this case, the arguments are fully specified in the shape, and this is a leaf operation.

Reply:
Done.